amical/tests/test_cli.py

```diff
@@ -12,6 +12,10 @@
 
 valid_commands = ["clean", "extract", "calibrate"]
 
+pytestmark = pytest.mark.filterwarnings(
+    "ignore: No SCI header for NIRISS. No PA correction will be applied."
+)
```

Comment:
This isn't idiomatic pytest. You're defining a decorator but it's not used anywhere. If you mean to filter a warning globally it should be done in `pyproject.toml:[tool.pytest]`.

Reply:
I took this from the pytest docs. That way it's ignored only in the specified module, and the warning is tested in `test_tools.py` so I don't want it ignored there.

![image](https://user-images.githubusercontent.com/35547958/181516461-a2c15ee8-f130-4683-8a34-6ef815e902c2.png)

Link: https://docs.pytest.org/en/7.1.x/how-to/capture-warnings.html

Reply:
My mistake, I shouldn't have been so assertive. Still, I'm not sure that ignoring the warning at the module level is a good idea. Are all functions tested here expected to trigger it ? If so, a comment could help :)

Reply:
Not all functions, I can handle it on a case-by-case basis if it is better this way. The reason I used the module-wide ignore was that I don't see a case where we don't want it ignored there, because this specific warning is tested in a different module, so if the tests pass there any instance warning in other module can be ignored. Maybe ignoring each instance explictely would be safer? Let me know if I should do this instead.

Reply:
I should speak my real concern more explicitly: is this warning expected to trigger often in production ? I don't mind ignoring it in specific tests but if users are going to see it all the time they may just stop trusting us with warnings and stop reading them (or even search for ways to turn _all_ python warnings off).

Reply:
It will trigger if you use the NIRISS instrument but there is no header available to perform PA correction. For real NIRISS data, this means that the position of a companion on sky would be off by > 100 deg. For simulated data with the official JWST simulation tools (MIRAGE) the same applies. For custom simulations (like the example data in AMICAL) the warning will trigger as well if the instrument is set to NIRISS, even if the real JWST coordinates are  not use (i.e. no PA correction is required).

Reply:
Okay that makes sense, thank you very much for this detailed explanation. Can you add a comment on the test module to make it clear that our test data doesn't contain this information and doesn't need it ?

Reply:
Sure :) Should I also put the warning for every file or do I keep the module-wide ignore?

Reply:
You can keep it at the module level, it's not a problem as long as there's a comment next to it :)

Reply:
I added the comment. Marking as resolved. Thanks!